aws/load_balancer/route53.tf

```diff
@@ -70,9 +70,9 @@ resource "aws_route53_record" "form_viewer_certificate_validation" {
 }
 
 resource "aws_route53_health_check" "gc_forms_application" {
-  fqdn              = var.domains[0]
-  port              = "443"
-  type              = "HTTPS"
+  fqdn              = aws_lb.form_viewer.dns_name
```

Comment:
Unfortunately I don't think this will work.  If you try to reach the lb dns routable name from the browser it's blocked by the WAF.  It returns a 403. 
We'll need to find another way...